src/pudl/output/ferc1.py

```diff
@@ -906,10 +912,44 @@ class NodeId(NamedTuple):
     plant_function: str | pandas_NAType
 
 
+@asset
+def _out_tags_ferc1(table_dimensions_ferc1) -> pd.DataFrame:
```

Comment:
I'm still getting oriented to them but I think we might call this `_out_ferc1__explosion_tags()` under the new naming conventions? Does that look right to you @bendnorman?

Reply:
`_out_ferc1__explosion_tags` looks good to me! Just to double check, this is not intended to be written to the database correct? Also, I included some criteria for when to create an intermediate asset as oppose to just keeping it as a function that can be called in a user facing asset:

>Reasons to create intermediate assets:
> - Long-running processing steps. It is convenient to separate the long and short-running processing portions of a dataset into separate assets so debugging the short-running process doesn’t take forever.
> - Steps that frequently need to be inspected when developing. For example, the _core_eia860 assets don’t take a long time to run, but it’s helpful to have them as separate assets so we can inspect the data before it is fed into the harvesting process.

Reply:
Oh also, do you think this asset deserves as `asset_type` portion? For example, do you think it's important to include the report frequency or a word that describes how the data is modeled?

Reply:
Yeah, this isn't meant to get written to the DB. It's constructed based on a manually compiled CSV of metadata that categorizes various values reported in the FERC 1, e.g. whether they should contribute to the utility's rate base or not. This is then programmatically filled in (so we don't have to enumerate every single combination of all of the dimensions). It's fast to build, but because it's manually compiled and then programmatically filled... probably a bit flaky and might need inspection as we go -- though maybe with better validation criteria that could be reduced.

There's nothing like report frequency, but the metadata is manually compiled on the basis of `table_name`, `xbrl_factoid`, `utility_type`, `plant_status`, and `plant_function` (the XBRL dimensions we're working with so far). Not sure what if any of that information would be useful to try and convey in the asset name.